File: 7주차/YH/fizzBuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        
        List = []
        
        for i in range(n) : 
            idx = i + 1
            if idx % 3 == 0 and idx % 5 == 0 : List.append("FizzBuzz")
            elif idx % 3 == 0 : List.append("Fizz")
            elif idx % 5 == 0 : List.append("Buzz")
            else : List.append(str(idx))
                
        return List

File: 07주차/NH/Fizz Buzz.js
function fizzBuzz(n) {
    let sample=[];
    for(let i=1; i<=n; i++)sample.push(String(i));
    for(let j=0; j<sample.length; j++){
        if(sample[j]%3==0&&sample[j]%5==0){
            sample[j]="FizzBuzz";
        }
        else if(sample[j]%3==0){
            sample[j]="Fizz";
        }
        else if(sample[j]%5==0){
            sample[j]="Buzz";
        }
    }
  return sample;
};